### PR TITLE
test(protocol): improve SGX tests

### PR DESCRIPTION
**Pull request by smtmfft**
1. remove useless testcase
2. make verify pass
3. add abi encoding test

### CODE REVIEW

File: packages/protocol/contracts/thirdparty/onchainRA/AutomataDcapV3Attestation.sol [hunk 1]
```diff
@@ -134,7 +134,8 @@ contract AutomataDcapV3Attestation is IAttestation {
     {
         return status == TCBInfoStruct.TCBStatus.OK
```
Comment (dantaik) [inline packages/protocol/contracts/thirdparty/onchainRA/AutomataDcapV3Attestation.sol:135]: @smtmfft please make improvements here: https://github.com/taikoxyz/automata-dcap-v3-attestation

Comment (dantaik) [reply packages/protocol/contracts/thirdparty/onchainRA/AutomataDcapV3Attestation.sol:135]: In the future, please release https://github.com/taikoxyz/automata-dcap-v3-attestation then in taiko-mono, we can use a "tag = v.1.0.0" to specify what code to use.

**Review by dantaik — APPROVED** [review]

**Review by adaki2004 — DISMISSED** [review]

**Comment by adaki2004** [comment]
Can you resolve conflicts ? Tests are failing too.

**Comment by smtmfft** [comment]
PR code is already in latest branch, close.

**Pull request closed without merging**